### DISCUSSION TITLE
Transaction failed (tutorial sdk)

### DISCUSSION THREAD

**Discussion by ylish** [aptos-labs/aptos-developer-discussions#127] [Questions] [answered]
Hello! I'm using Rust-sdk and trying to implement code from tutorial https://aptos.dev/tutorials/your-first-transaction .
And result is error in transaction. What am I doing wrong?
![t (5)](https://github.com/aptos-labs/aptos-developer-discussions/assets/17229178/1a09db9b-2369-4e6f-9a81-35dd0688cb10)

**Comment by hariria** [comment, marked as answer]
Hmm, your system clock (the clock on your PC / computer) may not be correct, causing the transaction to expire immediately, thus why you're receiving the `TRANSACTION_EXPIRED` error. Could you double check if that's the case?

**Reply by ylish** [reply]
You've hit the mark! That was the reason. Thank you very much.
